apps/src/templates/CodeWorkspace.jsx

```diff
@@ -48,7 +49,7 @@ var CodeWorkspace = React.createClass({
     readonlyWorkspace: React.PropTypes.bool.isRequired,
     showDebugger: React.PropTypes.bool,
     isRunning: React.PropTypes.bool,
-    isMinecraft: React.PropTypes.bool
+    isMinecraft: React.PropTypes.bool.isRequired
```

Comment:
Nit: Could also make `editCode`, `readOnlyWorkspace`, `isRunning`, `showDebugger` required if you also boolean-coerce them in your connect function.